diff --git a/dist/server/lib/start-server.js b/dist/server/lib/start-server.js
index 51607cdf365091d9e0fc11f14e8b5b3b6824357d..5e4ea6cfc14e942c599a469eeb8cc8aceace8e45 100644
--- a/dist/server/lib/start-server.js
+++ b/dist/server/lib/start-server.js
@@ -218,7 +218,7 @@ async function startServer(serverOptions) {
             const networkHostname = hostname ?? (0, _getnetworkhost.getNetworkHost)((0, _isipv6.isIPv6)(actualHostname) ? 'IPv6' : 'IPv4');
             const protocol = selfSignedCertificate ? 'https' : 'http';
             const networkUrl = networkHostname ? `${protocol}://${(0, _formathostname.formatHostname)(networkHostname)}:${port}` : null;
-            const appUrl = `${protocol}://${formattedHostname}:${port}`;
+            const appUrl = formattedHostname === 'localhost' && port.toString() === '3010' ? 'https://helperai.dev' : `${protocol}://${formattedHostname}:${port}`;
             if (nodeDebugType) {
                 const formattedDebugAddress = (0, _utils.getFormattedDebugAddress)();
                 _log.info(`the --${nodeDebugType} option was detected, the Next.js router server should be inspected at ${formattedDebugAddress}.`);
